addons/mail/static/src/models/rtc/rtc.js

```diff
@@ -757,8 +768,10 @@ registerModel({
                 peerConnection.close();
             }
             delete this._peerConnections[token];
-            browser.clearTimeout(this._fallBackTimeouts[token]);
-            delete this._fallBackTimeouts[token];
+            browser.clearTimeout(this.fallBackTimeouts[token]);
+            const newFallBackTimeouts = { ...this.fallBackTimeouts };
+            delete newFallBackTimeouts[token];
+            this.update({ fallBackTimeouts: newFallBackTimeouts });
```

Comment:
new model?

Reply:
Same remark as in `Rtc/rtcDataChannels` PR. If you are sure `token` is conceptually exactly the same as `RtcSession/id`, then can do some cleaning in model so that I can introduce `RtcSession/fallbackTimeout`.
It's pretty unsafe to do if we are unsure, because the current code only deal with single `RtcSession` from `Rtc` (`Rtc/currentRtcSession`), while `Rtc/rtcDataChannels` and `Rtc/fallBackTimeouts` deal with many tokens at the same time.

Reply:
Ok, will use `Rtc/rtcDataChannels` PR to test and see if it works as expected, and then apply same approach with `Rtc/fallBackTimeouts` (and maybe `Rtc/peerConnections`).